meta-refkit/classes/image-dsk.bbclass

```diff
@@ -227,6 +227,16 @@ python do_uefiapp_setscene () {
     sstate_setscene(d)
 }
 
+uefiapp_sign() {
+    if [ -e "${REFKIT_DB_KEY}" -a -e "${REFKIT_DB_CERT}" ] ; then
+        for i in `find ${DEPLOYDIR} -name '*.efi'`; do
+            sbsign --key ${REFKIT_DB_KEY} --cert ${REFKIT_DB_CERT} $i
+            sbverify --cert ${REFKIT_DB_CERT} $i.signed
+            mv $i.signed $i
+        done
+    fi
+}
```

Comment:
Does uefiapp_sign run again when the signing keys get changed? Probably not, because the dependency on the file content isn't specified. That may surprise developers.

For dm-verity I have solved this with: https://github.com/intel/intel-iot-refkit/blob/master/meta-refkit/classes/refkit-hash-dm-verity-key.bbclass